simple_visualizer_plugin/pyproject.toml

```
@@ -0,0 +1,23 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "simple_visualizer_plugin"
+author = [
+    {name = "Nikola Mitrović", email = "mitrovic.sv18.2021@uns.ac.rs"}
+]
+description = "Simple visualizer plugin for visualization of the graph"
+requires-python = ">=3.9"
+classifiers = [
+    "Programming Language :: Python :: 3",
+]
+dependencies = [
+    "graph_visualizer_api",
+    "os",
+    "jinja2",
+    "d3"
+]
+[project.entry-points."gv.plugins.visualizer"]
+simple = "simple_visualizer_plugin.visualizer:SimpleVisualiser"
```

There is a small typo, it should be SimpleVisualizer.

```suggestion
simple = "simple_visualizer_plugin.visualizer:SimpleVisualizer"
```